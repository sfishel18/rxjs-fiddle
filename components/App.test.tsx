import initStoryshots from '@storybook/addon-storyshots';
import React from 'react';

const mockReact = React;

jest.mock('../hooks/use-view-model', () => () => ({
  examples: [],
  submitEditorState: () => {},
}));

jest.mock('./CodeEditor', () => (props: {}) => mockReact.createElement('CodeEditor', props));
jest.mock('./OutputViz', () => (props: {}) => mockReact.createElement('OutputViz', props));

initStoryshots({
  storyKindRegex: /^App$/,
});
