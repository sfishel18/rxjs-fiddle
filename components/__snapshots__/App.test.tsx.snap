// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots App Default State 1`] = `
<div
  className="css-1k6vrxp"
>
  <div
    className="chakra-stack css-bb1f5k"
    style={
      Object {
        "height": "100%",
      }
    }
  >
    <section
      className="css-a6iqn1"
    >
      <header
        className="chakra-stack css-1v40tsk"
        style={
          Object {
            "marginBottom": "15px",
          }
        }
      >
        <button
          className="chakra-button css-1d88tv1"
          onClick={[Function]}
          type="button"
        >
          Run
        </button>
        <button
          aria-controls="menu-list-2"
          aria-expanded={false}
          aria-haspopup="menu"
          className="chakra-button chakra-menu__menu-button css-1d88tv1"
          id="menu-button-2"
          onClick={[Function]}
          onKeyDown={[Function]}
          type="button"
        >
          <span
            className="css-a9hmjk"
          >
            Examples
          </span>
          <span
            className="chakra-button__icon css-1f2k2gl"
          >
            <svg
              aria-hidden={true}
              className="chakra-icon css-onkibi"
              focusable={false}
              viewBox="0 0 24 24"
            >
              <path
                d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                fill="currentColor"
              />
            </svg>
          </span>
        </button>
        <mock-portal />
      </header>
      <div
        className="css-94739u"
      >
        <CodeEditor />
      </div>
    </section>
    <section
      className="css-1swlae3"
    >
      <OutputViz />
    </section>
  </div>
</div>
`;
